BitwardenShared/UI/Vault/VaultItem/LoginItemState.swift

```diff
@@ -10,6 +10,9 @@ struct LoginItemState: Equatable {
     /// Whether the user has permissions to view the cipher's password.
     var canViewPassword: Bool = true
 
+    /// Whether the user has permissions to edit the cipher
+    var editView: Bool = true
```

Comment:
❓ Does this only affect passkey deletion?

Reply:
For now yes, we only use that property to show/hide the remove passkey button.